src/operator/linalg_impl.h

```diff
@@ -519,7 +519,7 @@ void linalg_trmm<gpu, DType>(const Tensor<gpu, 2, DType>& A, const Tensor<gpu, 2
                             (rightside ? CUBLAS_SIDE_LEFT : CUBLAS_SIDE_RIGHT), \
                             (lower ? CUBLAS_FILL_MODE_UPPER : CUBLAS_FILL_MODE_LOWER), \
                             (transpose ? CUBLAS_OP_T : CUBLAS_OP_N), \
-                            CUBLAS_DIAG_NON_UNIT, B.size(0), B.size(1), &alpha, \
+                            CUBLAS_DIAG_NON_UNIT, B.size(1), B.size(0), &alpha, \
```

Comment:
was this a bug?

Reply:
Yes. Found it during testing the other operators